### PR TITLE
Component options - remove ambiguity

### PR DESCRIPTION
**Pull request by brianteeman**
Simple changes to clarify some of the option titles etc

### Smart Search - before
smartsearch-before

### Smart Search - after
smartsearch-after

### Search - before
search-before

### Search - after
search-after

### Media - before
media-before

### Media - after
media-after

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_search.ini [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ; Copyright (C) 2005 - 2016 Open Source Matters. All rights reserved.
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
-
+COM_SEARCH_FIELDSET_SEARCH_OPTIONS_LABEL="Smart Search"
```
Comment (infograf768) [inline administrator/language/en-GB/en-GB.com_search.ini]: ??

File: administrator/language/en-GB/en-GB.com_search.ini [hunk 2]
```diff
@@ -23,6 +23,7 @@ COM_SEARCH_FIELD_SEARCH_PHRASES_DESC="Show the search options."
 COM_SEARCH_FIELD_SEARCH_PHRASES_LABEL="Use Search Options"
 COM_SEARCH_FIELD_SEARCH_AREAS_DESC="Show the search areas checkboxes."
 COM_SEARCH_FIELD_SEARCH_AREAS_LABEL="Use Search Areas"
+COM_SEARCH_FIELDSET_SEARCH_OPTIONS_LABEL="Search"
```
Comment (infograf768) [inline administrator/language/en-GB/en-GB.com_search.ini]: Please alpha order.

File: administrator/language/en-GB/en-GB.com_search.ini (hunk 1 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_search.ini]: oopsie

File: administrator/language/en-GB/en-GB.com_search.ini (hunk 2 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_search.ini]: oops

**Comment by jeckodevelopment** [comment]
@brianteeman i've a question... wouldn't be better to say "Allowed Extensions (File Types)" instead of "Legal Extensions (File Types)"?

**Comment by brianteeman** [comment]
@jeckodevelopment its a specific technical term

**Comment by jeckodevelopment** [comment]
Thanks

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 46a1e52c6240975c3c1f5a6ef4fd707ea8b69b51<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12987">issues.joomla.org/tracker/joomla-cms/12987</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 46a1e52c6240975c3c1f5a6ef4fd707ea8b69b51<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12987">issues.joomla.org/tracker/joomla-cms/12987</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12987">issues.joomla.org/tracker/joomla-cms/12987</a>.</sub>

**Comment by rdeutz** [comment]
@brianteeman could you resolve :-) the conflicts, thanks

**Comment by brianteeman** [comment]
@rdeutz conflicts resolved

**Comment by rdeutz** [comment]
Bravo and without creating a new PR ;-)

**Comment by brianteeman** [comment]
Told you I would learn how

**Pull request merged**